src/js/tabs/firmware_flasher.js

```diff
@@ -977,13 +977,14 @@ TABS.firmware_flasher.initialize = function (callback) {
                     }
                 }
             }
+            $('a.exit_dfu').addClass('disabled');
```

Comment:
Two things here:
- using `$(this)` here instead of `$('a.exit_dfu')` will make it more portable;
- in general, this is not in the right place, as this will not disable the button if for example the user disconnects / connects the board to power cycle it - better hook into the port picker (`div#port-picker #port`), like this is done in the original pull request (#1863).

Reply:
Im working in this request change